docs/core/compatibility/unsupported-apis.md

```diff
@@ -0,0 +1,373 @@
+---
+title: Unsupported APIs on .NET Core
+description: Learn which APIs from the .NET Framework that always throw an exception on .NET Core.
+ms.date: 12/23/2019
+---
+# APIs that always throw exceptions on .NET Core
+
+The following APIs will always through a <xref:System.PlatformNotSupportedException> when run on .NET Core on the specified platform.
+
+This article organizes the affected API members by namespace.
+
+> [!NOTE]
+> This article is a work-in-progress. It is not a complete list of APIs that throw exceptions on .NET Core.
+
+## System
+
+> [!div class="mx-tdCol1BreakAll"]
+> | Member | Platform |
+> | - | - |
+> | <xref:System.AppDomain.CreateDomain%2A?displayProperty=nameWithType> | All |
+> | <xref:System.AppDomain.ExecuteAssembly(System.String,System.String[],System.Byte[],System.Configuration.Assemblies.AssemblyHashAlgorithm)?displayProperty=nameWithType> | All |
+> | <xref:System.Console.CapsLock?displayProperty=nameWithType> | Unix |
```

Comment:
I think it would be good to standardize on the platform description. Unix is the same as non-Windows. 

Also, we usually say macOS instead of OSX on docs.

Reply:
Good call.